internal/monitors/kubernetes/cluster/metrics/openshiftquotas.go

```diff
@@ -0,0 +1,72 @@
+package metrics
+
+import (
+	"fmt"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+
+	quota "github.com/openshift/api/quota/v1"
+	"github.com/signalfx/golib/datapoint"
+)
+
+// If additional metrics are added probably need to add additional resources here.
+var resources = []v1.ResourceName{
+	v1.ResourceCPU,
+	v1.ResourceMemory,
+	v1.ResourcePods,
+	v1.ResourceServices,
+	v1.ResourcePersistentVolumeClaims,
+	v1.ResourceServicesNodePorts,
+	v1.ResourceServicesLoadBalancers,
+}
+
+func datapointsForClusterQuotas(quota *quota.ClusterResourceQuota) []*datapoint.Datapoint {
+	dimensions := map[string]string{
+		"metric_source": "openshift",
+		"uid":           string(quota.UID),
```

Comment:
I think probably should be `kubernetes_uid` but will defer to @keitwb 
We use `uid` on daemonsets and replication controllers, but `kubernetes_uid` on deployments and replicasets. And pods have their own `kubernetes_pod_uid`.

Reply:
Should any of them use uid instead of kubernetes_uid? Is there a difference between them?

Reply:
Yes, let's stick with `kubernetes_uid`.